src/rgw/setup/rgw.py

```diff
@@ -544,7 +544,16 @@ def _update_hax_endpoint_and_create_admin(conf: MappedConf):
             #    Log.info(f'User creation is successful on "{Rgw._machine_id}" node.')
             #    Rgw._set_consul_kv(rgw_consul_idx, const.CONSUL_LOCK_KEY, const.ADMIN_USER_CREATED)
             # else:
-            machine_ids = Rgw._get_cortx_conf(conf, const.MACHINE_IDS_KEY)
+            storage_set = Rgw._get_cortx_conf(conf,\
+                f'node>{Rgw._machine_id}>storage_set')
```

Comment:
Could you move 'node>{Rgw._machine_id}>storage_set' to const.py and use the defined variable here

Reply:
updated in recent patch